### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by CodingCookieRookie**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -649,21 +649,26 @@ Adds a session to the session list.
...
-:bulb: **Tip:** Two sessions cannot have the same name but they can have the same date
+</div>
 
-Examples:
+:alert-warning: You will be exited from any currently entered session upon using the command.
+
+:green_book: Examples:
 * `addses s/tut1 d/12/7/2020`
 
+:bulb: **Tip:** Two sessions cannot have the same name but they can have the same date
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps these parts could be wrapped in their respective div boxes for better organisation of info?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -689,10 +689,18 @@ Delete INDEX? (yes/no)
...
    * If no: the specified session will not be removed from the session list.
 
-Examples:
+</div>
+
+:alert-warning: You will be exited from any currently entered session upon using the command.
+
+:green_book: Examples:
 * `deleteses 1` followed by `no` results in nothing changed.
 * `deleteses 2` followed by `yes` deletes the 2nd session in the session list.
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps these parts could be wrapped in their respective div boxes for better organisation of info? Noticed the same issue in other areas as well.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -719,8 +720,18 @@ Edit INDEX? (yes/no)
...
    * If no: the specified session will not be updated.
 
-Examples:
-* `editses 2 s/Tutorial1 d/10/10/2020` followed by `yes` edits the 2nd session in the session list with a new session name `Tutorial1` and a new session date `10/10/2020`.
+</div>
+
+:alert-warning: You will be exited from any currently entered session upon using the command.
+
+:green_book: Examples:
+* `editses 1 s/Tutorial 6 d/10/10/2020` followed by `yes` edits the 1st session in the session list with a new session name `Tutorial 6` and a new session date `10/10/2020`.
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps these parts could be wrapped in their respective div boxes for better organisation of info? Noticed the same issue in other areas as well.

**Review by erisjacey — COMMENTED** [review]
Do take note of some of my review comments wrt `div` usage (or lack thereof), I think it'll add to readability if it is used!

File: docs/UserGuide.md (hunk 3 above)
Comment (CodingCookieRookie) [reply docs/UserGuide.md]: Edited with thanks

**Review by marcustw — APPROVED** [review]
LGTM!

**Pull request merged**